intranet/static/js/eighth/schedule.js

```diff
@@ -37,133 +38,128 @@ $(function() {
-            var current_row = el.parent().parent();
-            var rows = [];
-            var row = current_row;
-            while(row && row.length > 0) {
+        var row;
+        var currentRow;
+        if (window.propagate_direction && window.propagate_direction === "down") {
+            currentRow = el.parent().parent();
+            rows = [];
+            row = currentRow;
+            while (row && row.length > 0) {
                 row = row.next();
-                if(row.hasClass("form-row")) {
+                if (row.hasClass("form-row")) {
                     rows.push(row[0]);
                 }
             }
             rows = $(rows);
         }
 
-        if(window.propagate_direction && window.propagate_direction == "up") {
-            var current_row = el.parent().parent();
-            var rows = [];
-            var row = current_row;
-            while(row && row.length > 0) {
+        if (window.propagate_direction && window.propagate_direction === "up") {
+            currentRow = el.parent().parent();
+            rows = [];
+            row = currentRow;
+            while (row && row.length > 0) {
                 row = row.prev();
-                if(row.hasClass("form-row")) {
+                if (row.hasClass("form-row")) {
                     rows.push(row[0]);
                 }
             }
             rows = $(rows);
         }
 
-        if(window.propagate_method && window.propagate_method == "only_new") {
+        if (window.propagate_method && window.propagate_method === "only_new") {
             var new_rows = [];
             for(var i=0; i<rows.length; i++) {
                 var rw = $(rows[i]);
-                if(!rw.hasClass("scheduled")) {
+                if (!rw.hasClass("scheduled")) {
                     new_rows.push(rows[i]);
                 }
             }
-            console.log(rows);
             rows = $(new_rows);
-            console.log(rows);
         }
 
-        if(el.hasClass("selectized")) {
-            console.info("Selectized element");
+        if (el.hasClass("selectized")) {
             var sel = el[0].selectize;
-            mod_items = sel.items;
-            console.info("New value:", mod_items);
+            var modItems = sel.items;
 
             rows.each(function() {
                 var ntd = $("td[data-field='" + field + "']", $(this));
                 var ninp = $("input, select", ntd);
                 var nsel = ninp[0].selectize;
                 var o_items = nsel.items;
-                nsel.setValue(mod_items);
-                console.debug("items: ", o_items, "=>", nsel.items);
+                nsel.setValue(modItems);
             });
         } else {
-            if(el.attr("type") == "checkbox") {
-                mod_val = el.prop("checked");
-                console.info("Checkbox new value:", mod_val);
+            if (el.attr("type") === "checkbox") {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected if as the only statement in an else block.](https://www.codacy.com/app/tjcsl/ion/file/2692319204/issues/source?bid=3176160&fileBranchId=3299251#l138)